### PR TITLE
Prepare for release 0.19.0

### PR DESCRIPTION
**Pull request by iketheadore**

### CODE REVIEW

File: src/gui/static/src/app/app.loadWallet.ts [hunk 1]
```diff
@@ -564,8 +570,8 @@ export class LoadWalletComponent implements OnInit {
     getDateTimeString(ts) {
         return moment.unix(ts).format("YYYY-MM-DD HH:mm")
     }
-    getElapsedTime(ts) {
-        return moment().unix() - ts;
+    getElapsedTime() {
+        return moment().unix() - this.blockChain[0].header.timestamp;
```
Comment (gz-c) [inline src/gui/static/src/app/app.loadWallet.ts]: can `blockChain.length` be 0 here?

Comment (gz-c) [inline src/gui/static/src/app/app.loadWallet.ts]: or is this method not necessary now that `elapsedTime` is set as a property

Comment (iketheadore) [inline src/gui/static/src/app/app.loadWallet.ts]: this function is not longer used

Comment (iketheadore) [inline src/gui/static/src/app/app.loadWallet.ts]: will remove this function

**Pull request merged**